packages/suite/src/views/wallet/send/components/Options/BitcoinOptions/Locktime.tsx

```diff
@@ -104,55 +90,57 @@ export const Locktime = ({ close }: LocktimeProps) => {
                 return translationString('LOCKTIME_IS_TOO_BIG');
             }
         },
-    });
+    };
+
+    const handleRbfSwitch = () => {
+        if (inputValue.length > 0) {
+            setValue(inputName, '');
+        }
+
+        toggleOption('bitcoinRBF');
+        composeTransaction(inputName);
+    };
 
     return (
-        <Wrapper>
-            <Input
+        <Card paddingType="large">
+            <StyledNumberInput
+                control={control}
+                name={inputName}
                 inputState={getInputState(error, inputValue)}
                 defaultValue={inputValue}
-                maxLength={MAX_LENGTH.BTC_LOCKTIME}
+                onChange={handleLocktimeChange}
+                rules={rules}
+                // maxLength={MAX_LENGTH.BTC_LOCKTIME}
```

Comment:
Is it intended?

Reply:
Yes, it is, I thought since the error message [is there](https://github.com/trezor/trezor-suite/blob/aa229eeeda83f939525df5d0c2adb012e22c3a26/packages/suite/src/views/wallet/send/components/Options/BitcoinOptions/Locktime.tsx#L89), there is no point in hard limiting the input + the native `maxLength` logic was counting commas in the NumberInput, so the `MAX_LENGTH.BTC_LOCKTIME` would not represent the length of the actual locktime value, further adding confusion.

But yeah, I forgot to clean up, done! 5e15588e4f